lib/scrabble.rb

```diff
@@ -1,5 +1,8 @@
 class Scrabble
-  def point_values
+  attr_reader :alphabet
+
+  def initialize
```

Comment:
You don't need to have an initialize - you could have just kept the point_values method and used its return value to access the hash!